test/AccountMgmt/AccountMgmtTest.php

```diff
@@ -132,18 +132,15 @@ public function testCreateAndDeleteKey() {
 
       // Delete Customer Key
       $delete_response = Traackr\AccountMgmt::customerkeyDelete(array('customer_key' => $customer_key));
```

Comment:
@bbguitar77 , looks like from these unit-tests that the delete endpoint used to return a JSON response of status->ok. Now it just returns empty-string. Not sure if you consider that a regression, but I've altered these tests to account for it. Go ahead and call the delete endpoint from the Livedocs page and you'll see what I mean.

Reply:
that's expected - when migrating the endpoints that don't explicitly need a body, I just been favoring not including a dummy "Ok" string in there. I'd like to switch the response code to a 202 for these put/delete operations, but had held off as unclear on whether the PHP client only considers a 200 a success (I think you patched that)

Reply:
Ok, if you look at the Bitbucket PR I tagged you on, you can see that I made the appropriate changes to just pass this blank string to the caller 🤷 

I think a 202 would work fine due to this code in TraackrApiObject::call():
```
        if (!($httpcode >= 200 && $httpcode <= 299)) {
            $info = curl_getinfo($this->curl);
...
```